src/handlers.rs

```diff
@@ -556,6 +567,46 @@ macro_rules! to_seconds {
   }};
 }
 
+fn handle_mkdir_sync(
+  _d: *const DenoC,
+  _builder: &mut FlatBufferBuilder,
+  _dirname: &str,
+  _perm: u32,
+) -> HandlerResult {
+  debug!("handle_mkdir_sync {}", _dirname);
+  let deno = from_c(_d);
+  if deno.flags.allow_write {
+    // TODO: apply permission
```

Comment:
Look at the code in #567 .

Reply:
Thanks for the reference! Have just added a similar way to set permission with `DirBuilderExt`